lib/services/event_service.ex

```diff
@@ -51,6 +51,20 @@ defmodule Aprb.Service.EventService do
                           ]
                         }]",
           unfurl_links: true }
+      "bidding" ->
+        %{
+          text: ":gavel: #{event["type"]} on #{event["lotId"]} from Paddle #{event["bidder"]["paddleNumber"]} ",
+          attachments: "[{
+                          \"fields\": [
+                            {
+                              \"title\": \"Amount\",
+                              \"value\": \"#{format_price(event["amountCents"] || 0)}\",
```

Comment:
Does this need to be divided by 100 before passing it to `format_price`? I see a Money and *100 in that method.